src/astt_gui/templates/index.html

```diff
@@ -202,15 +202,15 @@ <h1>CAM Dashboard</h1>
                   <div class="row">
                     <div class="col">
                      <h4>Stow-Pin State</h4>
-                     <button class="btn btn-secondary" style="width: 190px;" id="stowState">UNKNOWN</button>
+                     <button class="btn btn-secondary" title="This button displays our stow state" style="width: 190px;" id="stowState">UNKNOWN</button>
                     </div>
                     <div class="col">
                       <h4>Modes state</h4>
-                      <button class="btn btn-secondary" style="width: 190px;" id="modeState">UNKNOWN</button>
+                      <button class="btn btn-secondary" title="This button displays our modes state" style="width: 190px;" id="modeState">UNKNOWN</button>
                     </div>
                     <div class="col">
                       <h4>Functional State</h4>
-                      <button class="btn btn-secondary" style="width: 190px;" id="funcState">UNKNOWN</button>
+                      <button class="btn btn-secondary" title="This button displays our functional state" style="width: 190px;" id="funcState">UNKNOWN</button>
                     </div>
```

Comment:
These are buttons yes on the code but on the UX are displayed a grids so stating that  for e.g"This button displays our modes state" the user might expect that to be clickable for action, so I think we should reframe adding "button" on the hint for the 3 states

Reply:
Done!